analyzers/rspec/cs/S4581_c#.html

```diff
@@ -10,6 +10,8 @@ <h2>Noncompliant Code Example</h2>
 public void Foo()
 {
     var g = new Guid(); // Noncompliant - what's the intent?
+    var guid = default(Guid);
```

Comment:
I would not update this, as we do not raise issue for these two cases. And I assume we should also create an issue for that.